.dockerignore

```
@@ -0,0 +1,6 @@
+.env
+.git
+.venv
+Dockerfile
+.vscode
+README.md
```

An extra folder I might consider adding:
```suggestion
README.md
**/__pycache__
```
(Note the `**/` at the start is required to ignore subfolders, unlike `.gitignore` which ignores both folders in the root and sub-folders automatically)